ctam/interfaces/fw_update_ifc.py

```diff
@@ -253,6 +224,14 @@ def ctam_stage_fw(
             else:
                 StageFWOOB_Status = True
         else:
+            error_check = JSONData.get("error", None)
+            if error_check and image_type != "large":
+                message = JSONData.get("error", {}).get("message", "")
+                if message.lower() != "An update is in progress.".lower():
```

Comment:
Just an observation - OCP spec has this message as "Another firmware update already in progress" (Table 11), but we see "An update is in progress." in Nvidia systems as well.

Reply:
Added one config file for redfish messages. We can add custom messages and then we can read and compare accordingly.